linux_os/guide/system/accounts/accounts-physical/require_emergency_target_auth/bash/cis_rhel7_require_emergency_target_auth.fail.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
I think we don't need another fail scenario, there is already one and we can make it applicable to all RHEL versions by modifying `platforms` metadata to contain `multi_platform_rhel`. (pass scenarios still need to be separate for RHEL7 and RHEL8).

Reply:
Pretty much same applies also to the `require_singleuser_auth` rule.